desktop/release-notes.md

```diff
@@ -24,11 +24,56 @@ Take a look at the [Docker Public Roadmap](https://github.com/docker/roadmap/pro
 
 For frequently asked questions about Docker Desktop releases, see [FAQs](faqs/general.md/#releases)
 
+## 4.20.0
+
+{% include release-date.html date="2023-05-25" %}
+
+{% include desktop-install.md all=true version="4.20.0" build_path="/" %}
+
+### New
+
+- [Docker Scout CLI](https://docs.docker.com/scout/#docker-scout-cli) find the most recently built image if none has been provided as an argument.
+
+### Upgrades
+
+- [Buildx v0.10.5](https://github.com/docker/buildx/releases/tag/v0.10.5)
+- [Compose v2.18.1](https://github.com/docker/compose/releases/tag/v2.18.1)
+- [Docker Engine v23.0.5](https://docs.docker.com/engine/release-notes/23.0/#2305)
+- [Containerd v1.6.21](https://github.com/containerd/containerd/releases/tag/v1.6.21)
+- [runc v1.1.7](https://github.com/opencontainers/runc/releases/tag/v1.1.5)
+- [runc v1.1.6](https://github.com/opencontainers/runc/releases/tag/v1.1.6)
+
+### Bug fixes and enhancements
+
+#### For all platforms
+
+- Added a warning about the [retirement of Docker Compose ECS/ACS integrations in November 2023](https://docs.docker.com/go/compose-ecs-eol/). Can be suppressed with `COMPOSE_CLOUD_EOL_SILENT=1`.
+- [Docker Scout CLI](https://docs.docker.com/scout/#docker-scout-cli) find the most recently built image if none has been provided as an argument.
+- Improved [Docker Scout CLI](https://docs.docker.com/scout/#docker-scout-cli) `compare` command.
```

Comment:
```suggestion
- Improved the [Docker Scout CLI](https://docs.docker.com/scout/#docker-scout-cli) `compare` command.
```

Reply:
this line I've also moved it to the new section. It's more of a personal opinion here to put Scout CLI in the _new_ section! So let me know if I should move it to the _bug fixes and enhancements_ section!

Reply:
I think we should keep it in Bug fixes and enhancements. Looking at previous releases, we've generally kept all the 'improved..' notes, in this section and it would be good to keep this consistent